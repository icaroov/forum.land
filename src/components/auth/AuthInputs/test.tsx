import { render, screen } from '@testing-library/react'

import AuthInputs from '.'

describe('<AuthInputs />', () => {
  it('should render the heading', () => {
    const { container } = render(<AuthInputs />)

    expect(
      screen.getByRole('heading', { name: /AuthInputs/i })
    ).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
