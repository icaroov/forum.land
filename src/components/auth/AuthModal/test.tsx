import { render, screen } from '@testing-library/react'

import AuthModal from '.'

describe('<AuthModal />', () => {
  it('should render the heading', () => {
    render(<AuthModal />)

    expect(
      screen.getByRole('heading', { name: /AuthModal/i })
    ).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
