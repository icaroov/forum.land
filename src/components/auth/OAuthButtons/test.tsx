import { render, screen } from '@testing-library/react'

import OAuthButtons from '.'

describe('<OAuthButtons />', () => {
  it('should render the heading', () => {
    render(<OAuthButtons />)

    expect(
      screen.getByRole('heading', { name: /OAuthButtons/i })
    ).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
