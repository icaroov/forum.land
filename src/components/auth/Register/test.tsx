import { render, screen } from '@testing-library/react'

import Register from '.'

describe('<Register />', () => {
  it('should render the heading', () => {
    const { container } = render(<Register />)

    expect(screen.getByRole('heading', { name: /Register/i })).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
