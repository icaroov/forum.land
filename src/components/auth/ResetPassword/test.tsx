import { render, screen } from '@testing-library/react'

import ResetPassword from '.'

describe('<ResetPassword />', () => {
  it('should render the heading', () => {
    const { container } = render(<ResetPassword />)

    expect(screen.getByRole('heading', { name: /ResetPassword/i })).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
