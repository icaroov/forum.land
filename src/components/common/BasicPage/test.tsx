import { render, screen } from '@testing-library/react'

import BasicPage from '.'

describe('<BasicPage />', () => {
  it('should render the heading', () => {
    const { container } = render(<BasicPage />)

    expect(
      screen.getByRole('heading', { name: /BasicPage/i })
    ).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
