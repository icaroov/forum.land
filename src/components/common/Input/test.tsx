import { render, screen } from '@testing-library/react'

import Input from '.'

describe('<Input />', () => {
  it('should render the heading', () => {
    render(<Input />)

    expect(screen.getByRole('heading', { name: /Input/i })).toBeInTheDocument()

    // expect(container.firstChild).toMatchSnapshot()
  })
})
