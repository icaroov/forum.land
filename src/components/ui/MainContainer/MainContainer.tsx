import { ChakraProvider } from '@chakra-ui/react'

import theme from '@styles/theme'

type MainContainerProps = {
  cookies?: string | undefined
  children: React.ReactNode
}

const MainContainer = ({ children }: MainContainerProps) => {
  return <ChakraProvider theme={theme}>{children}</ChakraProvider>
}

export default MainContainer
