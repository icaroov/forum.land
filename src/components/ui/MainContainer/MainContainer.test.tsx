import { screen } from '@testing-library/react'

import { renderWithProviders } from '@src/config/test/renderWithProviders'

import MainContainer from '.'

describe('<MainContainer />', () => {
  it('should render the component correctly', () => {
    const { container } = renderWithProviders(
      <MainContainer cookies="fake-cookies">
        <h1>Test</h1>
      </MainContainer>
    )

    expect(container).toMatchSnapshot()
    expect(screen.getByText('Test')).toBeInTheDocument()
  })
})
