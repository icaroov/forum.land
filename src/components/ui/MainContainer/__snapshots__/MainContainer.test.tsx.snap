// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContainer /> should render the component correctly 1`] = `
<div>
  <h1>
    Test
  </h1>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
